config.py

```diff
@@ -74,7 +74,7 @@ class Test(Config):
 
     DM_NOTIFY_API_KEY = "not_a_real_key-00000000-fake-uuid-0000-000000000000"
     SHARED_EMAIL_KEY = "KEY"
-    SECRET_KEY = "KEY"
+    SECRET_KEY = "KEY2"
```

Comment:
I'm guessing this is just to differentiate between the two keys for tests? Make sure no confusion?

Reply:
Yes in that there should be no confusion in a developers understanding that the two are different but also so that our tests don't risk being incorrect. We don't want our
tests accidentally passing if we were accidentally using one of the keys instead of the other but we wouldn't know because their values are the same.